code/model/modelinterp.cpp

```diff
@@ -3641,9 +3655,22 @@ void submodel_get_two_random_points_better(int model_num, int submodel_num, vec3
 			return;
 		}
 
+#ifndef NDEBUG
+		if (RAND_MAX < nv)
+		{
+			static int submodel_get_two_random_points_warned = false;
+			if (!submodel_get_two_random_points_warned)
+			{
+				polymodel *pm = model_get(model_num);
```

Comment:
This line's the one being complained about; since it's set to the same thing on line 3636, this line can just be removed outright.